phpcs.ruleset.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset name="GiveWP Coding Standards">
+	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
+
+	<!-- Set a description for this ruleset. -->
+	<description>We love clean code for GiveWP. This will let us enforce that easily.</description>
+
+	<!-- Exclude directories we don't want checked. -->
+	<exclude-pattern>node_modules/*</exclude-pattern>
+	<exclude-pattern>vendor/*</exclude-pattern>
+	<exclude-pattern>sample-data/*</exclude-pattern>
+	<exclude-pattern>languages/*</exclude-pattern>
+	<exclude-pattern>resources/css/*</exclude-pattern>
+	<exclude-pattern>resources/js/*</exclude-pattern>
+	<exclude-pattern>resources/images/*</exclude-pattern>
+
+	<!-- Bring in WP rules. -->
+	<rule ref="WordPress-Core">
+		<exclude name="WordPress.Files.FileName.InvalidClassFileName" />
+		<exclude name="WordPress.WP.I18n.MissingTranslatorsComment" />
+		<exclude name="Generic.Arrays.DisallowShortArraySyntax" />
+	</rule>
+
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
+	  <type>warning</type>
+	</rule>
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<property name="text_domain" type="array" value="give"/>
```

Comment:
@alaca  can we text-domain dynamic

Reply:
Yes, I can add that file to the list.

Reply:
Is this resolved, @alaca?

Reply:
Yes, @JasonTheAdams.  
https://github.com/impress-org/givewp-addon-boilerplate/pull/20/commits/9a4124888779aa737ea31f53504131b7ab92664d